### ISSUE TITLE
"I forgot my password" functionality

### ISSUE THREAD

**Issue by mmbotelho** [okTurtles/group-income#569]
### Problem

At the moment, there's no way for a user to retrieve their password if they forget it. 

### Solution

Design this flow. It should look something like:
1. Click "I forgot my password";
2. Enter your email;
3. An email is sent with a link to reset the password;
4. Land in a "Reset your password" page;
5. Enter new password;
5. Success!

@taoeffect I remember there were some issues with sending emails through the platform. Can you please validate if something like this would be possible?

**Comment by taoeffect** [comment]
> Can you please validate if something like this would be possible?

Yeah, as mentioned in related comment on Slack, this is something we are making "Low Priority" for now because of the difficulty involved in getting this right.

1. Email as a protocol is in its final days, and we do not want to support its continued existence.
2. Email is extremely insecure. Group Income is supposed to be "secure" (by modern standards).
3. Email is a nightmare to integrate with.
4. The way we are handling user logins is radically different from how it's traditionally done.

For these and other reasons we most likely will not be interfacing with email — but a password reset is something we would like to support. Just not like this.
